packages/source-iottwinmaker/src/scene-module/SceneMetadataModule.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { IoTTwinMakerClient } from '@aws-sdk/client-iottwinmaker';
+import { SceneMetadataModule } from './SceneMetadataModule';
+import { SecretsManagerClient } from '@aws-sdk/client-secrets-manager';
+
+/*eslint-disable @typescript-eslint/no-explicit-any */
+
+class MockResponse {
+  data: any;
+
+  constructor(data: any) {
```

Comment:
no any

Reply:
Not required after refactoring - Removed the code